components/automate-ui/src/app/entities/orgs/org.selectors.ts

```diff
@@ -43,5 +43,5 @@ export const deleteStatus = createSelector(
 export const orgFromRoute = createSelector(
   orgEntities,
   routeParams,
-  (state, { orgid }) => state[orgid]
+  (state, { 'org-id': orgid }) => state[orgid]
```

Comment:
Prefer that multiple words are always distinguished in some fashion, perhaps this for consistency with "cookbook_name":
```suggestion
  (state, { 'org-id': org_id }) => state[org_id]
```